README.md

```diff
@@ -1,27 +1,72 @@
 # Canonical Webteam Flask-Base
```

Comment:
Could you add a little note on how to use poetry-setup in here?

Reply:
Yep will do